components/match2/commons/match_group_legacy.lua

```diff
@@ -301,7 +302,7 @@ function MatchGroupLegacy:getMatch(match2key, match1params)
 			return nil
 		end
 	end
-	return match
+	return Match.makeEncodedJson(match)
```

Comment:
why?

Reply:
Maybe not need if I move this https://github.com/Liquipedia/Lua-Modules/blob/c0c61a6b68a849461ca9ad1c407a352f5f5f1871/components/match2/wikis/ageofempires/match_group_input_custom.lua#L38 and place it after opponents have been processed

Reply:
![image](https://github.com/user-attachments/assets/c9d995b8-37c2-4c96-ad45-35e8818c5ac2)
when I move match.mode it complains about something else xD,